### DISCUSSION TITLE
V4 beta 5 - is there a newer firmware available?

### DISCUSSION THREAD

**Discussion by roboengr** [forkineye/ESPixelStick#629] [General]
**ESPixelStick Firmware Version:** v4 beta 4
**Hardware Version:** D1 mini 8266(esp8266MOD chip) with one relay output
**Binary release** used.
**Operating System (and version)** win 10 latest
**Web Browser** (and version);chrome latest
**Access Point:** Ubiquiti NanoStation locoM2 
**Describe the bug:** crash and status settings are opposite,  turn off dynamic and it loads dynamic.  turn on dynamic  it load static.
The loader jar seems to be defective.  I am trying to use the firmware just the way it is down loaded.  it seems to be not selecting the d1 mini 8266 for some reason.

tried all betas with different results from each all not working. there was talk about relay fixes GPIO -1 compared to zero. and other modifications.  should there not be a new version out now since nov 21 2022?  where do we get the latest version?

**Comment by roboengr** [comment]
will the v4 beta 4 not work on this device below?

![esp8266mod d1 mini](https://user-images.githubusercontent.com/8006653/228431317-34e7f9ad-d6ff-41fd-b81a-92dc4a4ea0d1.jpg)

**Comment by roboengr** [comment]
Martin, thank you very very much for all the work you have done on this project.

Will the v4 beta 4  work on a esp8266 dev board?  or is the esp32 the only boards it will work on?  Please explain what has been tested?

If possible give a list of steps to load?  
1. edit the config file with the ip address, GW and mask, with static and reboot settings set to true(What file)
2. run Jar
3. press download button (which button???)
4. make sure the top text windows shows complete
5. make sure bottom window shows the ip address and connection to a static server( how to tell and what should we see???)
6. pull up on web browser
7. set the configuration(how to set up ws2812 and one relay output???)

what java version?  what other versions of things do I need on a win 10 h2 system?

the home Wiki does not tell anything about it, what chips where we should buy from and type is best..  Can we get more information added?

**Comment by forkineye** [comment]
You can grab the latest CI builds - https://github.com/forkineye/ESPixelStick/actions/workflows/build.yaml.  Just click on the newest entry and scroll down to "Release Archive" under "Artifacts".

**Comment by MartinMueller2003** [comment]
The latest V4 builds support multiple ESP8266 platforms as well as a rich set of ESP32 platforms. It even has a version for ESP32 D1 Mini running on the ESPv3 hardware. Please take a look at the supported platforms list.

**Comment by MartinMueller2003** [comment]
You can try the latest builds here: https://espixelstickwebflasher.from-ct.com:5000/
